armi/resources/referenceFissionProducts.dat

```diff
@@ -1,719 +1,717 @@
-13          LFP41 TE1257 7.0000E-05
-13          LFP41 TE1267 2.0000E-05
-13          LFP41 TE27M7 2.8000E-04
-13          LFP41 TE1287 5.2000E-03
-13          LFP41 TE29M7 1.7900E-03
-13          LFP41 TE1307 1.5340E-02
-13          LFP41 TE1327 9.4000E-04
-13          LFP41 I127_7 2.7700E-03
-13          LFP41 I129_7 6.6400E-03
-13          LFP41 I131_7 1.5000E-03
-13          LFP41 XE1287 7.0000E-05
-13          LFP41 XE1307 1.1000E-04
-13          LFP41 XE1317 2.6830E-02
-13          LFP41 XE1327 4.4050E-02
-13          LFP41 XE1337 1.9100E-03
-13          LFP41 XE1347 6.1020E-02
-13          LFP41 XE1367 7.1120E-02
-13          LFP41 CS1337 5.2310E-02
-13          LFP41 CS1347 1.2100E-03
-13          LFP41 CS1357 6.7050E-02
-13          LFP41 CS1367 5.0000E-05
-13          LFP41 CS1377 7.1810E-02
-13          LFP41 BA1347 8.0000E-05
-13          LFP41 BA1367 3.1000E-04
-13          LFP41 BA1377 5.2000E-04
-13          LFP41 BA1387 6.7380E-02
-13          LFP41 BA1407 4.6100E-03
-13          LFP41 LA1397 6.0930E-02
-13          LFP41 LA1407 6.1000E-04
-13          LFP41 CE1407 5.0310E-02
-13          LFP41 CE1417 1.0740E-02
-13          LFP41 CE1427 4.9900E-02
-13          LFP41 CE1437 5.0000E-04
-13          LFP41 CE1447 3.3360E-02
-13          LFP41 PR1417 4.0060E-02
-13          LFP41 PR1437 4.8100E-03
-13          LFP41 ND1427 2.2000E-04
-13          LFP41 ND1437 4.8540E-02
-13          LFP41 ND1447 1.0750E-02
-13          LFP41 ND1457 3.4770E-02
-13          LFP41 ND1467 3.0510E-02
-13          LFP41 ND1477 1.7600E-03
-13          LFP41 ND1487 1.9290E-02
-13          LFP41 ND1507 1.2900E-02
-13          LFP41 PM1477 1.9790E-02
-13          LFP41 PM1487 5.0000E-05
-13          LFP41 PM48M7 2.3000E-04
-13          LFP41 PM1497 2.4000E-04
-13          LFP41 SM1477 1.5500E-03
-13          LFP41 SM1487 9.6000E-04
-13          LFP41 SM1497 1.5200E-02
-13          LFP41 SM1507 1.0900E-03
-13          LFP41 SM1517 9.1100E-03
-13          LFP41 SM1527 9.3000E-03
-13          LFP41 SM1537 9.0000E-05
-13          LFP41 SM1547 4.4700E-03
-13          LFP41 EU1537 5.7400E-03
-13          LFP41 EU1547 6.1000E-04
-13          LFP41 EU1557 2.9500E-03
-13          LFP41 EU1567 3.0000E-04
-13          LFP41 GD1547 1.0000E-05
-13          LFP41 GD1557 1.2000E-04
-13          LFP41 GD1567 2.4600E-03
-13          LFP41 GD1577 1.3900E-03
-13          LFP41 GD1587 1.3500E-03
-13          LFP41 GD1607 3.9000E-04
-13          LFP41 TB1597 6.2000E-04
-13          LFP41 TB1607 2.0000E-05
-13          LFP41 DY1607 2.0000E-05
-13          LFP41 DY1617 2.0000E-04
-13          LFP41 DY1627 2.0000E-05
-13          DUMP1 DUMMY1 1.0000E-06
-13          DUMP2 DUMMY1 1.0000E-06
\ No newline at end of file
+LFP35 GE76   1.5000E-04
+LFP35 AS75   7.0000E-05
+LFP35 SE77   2.9000E-04
+LFP35 SE78   5.6000E-04
+LFP35 SE79   9.1000E-04
+LFP35 SE80   1.5100E-03
+LFP35 SE82   3.9000E-03
+LFP35 BR81   2.4600E-03
+LFP35 KR82   6.0000E-05
+LFP35 KR83   6.0100E-03
```

Comment:
This is just a formatting change. The `13` input on the first two columns is actually CARD 13 from `MC2-2` and there is no reason to keep it around. Additionally, I updated the nuclide labels to be based on the nuclides that were implemented/updated in #998 rather than being the `MC2-3` labels.